src/Animals.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Animals {
+
+        protected String name;
+        protected int age;
+        protected static int count = 0;
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.